CHANGELOG.md

```diff
@@ -2,8 +2,25 @@
 
 ## unreleased
 
+## 3.0.0-beta.1 / 2024-09-25
+
+* [CHANGE] regexp `.` now matches all characters (performance improvement). #14505
 * [CHANGE] `holt_winters` is now called `double_exponential_smoothing` and moves behind the [experimental-promql-functions feature flag](https://prometheus.io/docs/prometheus/latest/feature_flags/#experimental-promql-functions). #14930
+* [CHANGE] Remove `no-default-scrape-port` feature flag. #14160
```

Comment:
Maybe be more explicit and say that now Prometheus will never add or remove the default port.